tests/apps/resiliencyapp_grpc/Dockerfile-windows

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM golang:1.17-nanoserver
```

Comment:
Will this work with the exact Windows Server version we need? Let's see in the run.

Reply:
I copied the existing grpc proxy test. So if this doesn't work, that shouldn't either.